src/main/java/ecdar/Ecdar.java

```diff
@@ -298,28 +281,26 @@ public void start(final Stage stage) {
         }));
 
         stage.setOnCloseRequest(event -> {
-            BackendHelper.stopQueries();
-
+            int status = 0;
             try {
-                backendDriver.closeAllEngineConnections();
-            } catch (IOException e) {
-                e.printStackTrace();
+                BackendHelper.clearEngineConnections();
+            } catch (BackendException e) {
+                // -1 indicates that an exception was thrown
+                status = -1;
```

Comment:
I think we should consider making a variable for ``-1`` and maybe call it ``ClosingBackendConnectionsFailedStatusCode``. Maybe ``0``(from ``int status = 0;`` should also have a variable called something like ``ShutdownSuccessfulStatusCode``.